src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt

```diff
@@ -117,14 +117,15 @@ class SnykToolWindowPanel(val project: Project) : JPanel(), Disposable {
                 override fun scanningCliError(cliError: CliError) {
                     currentCliResults = null
                     ApplicationManager.getApplication().invokeLater {
+                        SnykBalloonNotifications.showError(cliError.message, project)
                         if (cliError.message.startsWith("Authentication failed. Please check the API token on ")) {
+                            getApplicationSettingsStateService().token = null
```

Comment:
Why do we want to remove existing token? Sure, it's invalid because of auth error, but e.g. endpoint can be changed.

Reply:
We have checks like `token != null` in various places (for example SnykRunScanAction#update, AdvisorClient creation. etc) to enable/allow some action to perform. And if we keep invalid token then many actions wil produce errors or have undefined behaviour.